Packages/vcs/Lib/editors/label.py

```diff
@@ -140,9 +150,19 @@ def is_object(self, label):
         return self.label == label
 
     def in_bounds(self, x, y):
-        t = self.get_text()
```

Comment:
`inside_label` was a nasty piece of work that has thankfully been replaced with the wonderful `vtkPropPicker`. Seriously, I totally wish I had discovered that thing earlier. So much pain and agony could have been saved.